.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn.lock
+.idea
```

Comment:
Don't make unrelated changes.

Reply:
This is my IDEs config files, happy to remove

Reply:
Yes. It's **your** config. It has nothing to do in a repo. It should be in your own global gitignore.

Reply:
Ok